fs/ensure_symlink_test.ts

```diff
@@ -238,7 +238,10 @@ Deno.test("ensureSymlinkSync() creates symlink with relative target", function (
   Deno.removeSync(testDir, { recursive: true });
 });
 
-Deno.test("ensureSymlink() works with URLs", async () => {
+Deno.test("ensureSymlink() works with URLs", {
+  // TODO(kt3k): The 2nd test case doesn't pass on Windows. Fix it.
```

Comment:
Able to create an issue?

Reply:
Created https://github.com/denoland/deno_std/issues/5092